upup/models/cloudup/resources/addons/networking.kuberouter/k8s-1.12.yaml.template

```diff
@@ -133,32 +128,34 @@ metadata:
   name: kube-router
   namespace: kube-system
 ---
-# Kube-router roles
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: kube-router
   namespace: kube-system
 rules:
-  - apiGroups: [""]
+- apiGroups:
+  - ""
     resources:
       - namespaces
       - pods
       - services
       - nodes
       - endpoints
     verbs:
-      - get
       - list
+  - get
```

Comment:
indentation issue here? and the apiGroups changes above and below.

Reply:
You are right. Forgot to enable white space diff and the editor went wild. Thanks :)